include/swift/AST/DiagnosticsClangImporter.def

```diff
@@ -258,8 +258,16 @@ ERROR(failed_base_method_call_synthesis,none,
       "failed to synthesize call to the base method %0 of type %0",
       (ValueDecl *, ValueDecl *))
 
-ERROR(both_returns_retained_returns_unretained,none,
-      "%0 cannot be annotated with both swift_attr('returns_retained') and swift_attr('returns_unretained') attributes", (const clang::NamedDecl*))
+ERROR(both_returns_retained_returns_unretained, none,
+      "%0 cannot be annotated with both swift_attr('returns_retained') and "
+      "swift_attr('returns_unretained') attributes",
```

Comment:
@fahadnayyar I think we should suggest people to use the macro instead of directly using `swift_attr`. What are other similar diagnostics doing?

Reply:
rdar://138810362